docs/UserGuide.md

```diff
@@ -30,79 +30,110 @@ EssenMakanan is an app that keeps track of ingredients that a user has in the ki
+| Edit ingredient                                           | edit i/INGREDIENT_NAME n/NEW_NAME q/NEW_QUANTITY u/NEW_UNIT | edit i/bread n/toast q/3 u/pc                                                      |
+| Delete Recipe                                             | delete r/RECIPE_TITLE                                       |                                                                                    |
+| Delete Ingredient                                         | delete r/INGREDIENT_NAME                                    |                                                                                    | 
+| View all ingredients                                      | view i                                                      |                                                                                    |
+| View all recipes                                          | view r                                                      |                                                                                    |
+| View specific ingredient                                  | view i/INGREDIENT_NAME <br><br>view i/INGREDIENT_ID         | view i/bread <br><br>view i/1                                                      |
+| View specific recipe                                      | view r/RECIPE_ID                                            | view r/1                                                                           |
+| Filter recipe by ingredients                              | filter recipe i/INGREDIENT_NAME [i/...]                     | filter recipe i/chicken i/noodles                                                  |
+| Start recipe <br>(view missing ingredients from a recipe) | start RECIPE_TITLE <br><br>start RECIPE_ID                  | start dumpling noodles <br><br> start 1                                            |
+| Help <br>(list all commands available)                    | help                                                        | help                                                                               |
 
 
 ----
 ### System
 
-1. Viewing help – `help`
+#### Viewing help – `help`
 
     List all commands and the format and brief description of each command.
 
 ---
-### Recipes 
-2. View all recipes -  `view r`
+### Recipes
+#### View all recipes -  `view r`
 
    List all recipes available inside the app.
+---
 
-
-3. View specific recipe - `view r/RECIPE_ID`
+#### View specific recipe - `view r/RECIPE_ID`
 
    View the steps of a specified recipe with RECIPE_ID
   
     <br> Examples :
   
     * `view r/1` to show a recipe at index ‘1’ of the list.
+---
 
+#### Add new recipe - `add r/RECIPE_TITLE s/STEP_DESCRIPTION i/INGREDIENT_NAME,QUANTITY,UNIT`
 
-4. Add new recipe - `add r/RECIPE_NAME`
+- `add r/RECIPE_TITLE t/TAG_NAME s/STEP_DESCRIPTION d/DURATION`
```

Comment:
You may want to state the different tags available or refer the user to another section.